File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/AttributeValueStateEventId.java
package org.dddml.wms.domain;


public class AttributeValueStateEventId
{
    private String attributeId;

    public String getAttributeId()
    {
        return this.attributeId;
    }

    public void setAttributeId(String attributeId)
    {
        this.attributeId = attributeId;
    }

    private String value;

    public String getValue()
    {
        return this.value;
    }

    public void setValue(String value)
    {
        this.value = value;
    }

    private Long attributeVersion;

    public Long getAttributeVersion()
    {
        return this.attributeVersion;
    }

    public void setAttributeVersion(Long attributeVersion)
    {
        this.attributeVersion = attributeVersion;
    }

}

File: Dddml.Wms.AdminUI/ts/AttributeValueStateEventId.ts
namespace Dddml.Wms.Domain {

	export class AttributeValueStateEventId {

		private attributeId: string;

		getAttributeId(): string {
			return this.attributeId;
		}

		setAttributeId(attributeId: string) {
			this.attributeId = attributeId;
		}

		private value: string;

		getValue(): string {
			return this.value;
		}

		setValue(value: string) {
			this.value = value;
		}

		private attributeVersion: string;

		getAttributeVersion(): string {
			return this.attributeVersion;
		}

		setAttributeVersion(attributeVersion: string) {
			this.attributeVersion = attributeVersion;
		}

	}
}

